content/rollbear/index.md

```diff
@@ -0,0 +1,18 @@
+---
+{
+	name: "Björn Fahller",
+	firstName: "Björn",
+	lastName: "Fahller",
+	description: "I write code, I do not drink coffee.\nAvatar by Franzisca Panter",
+	socials: {
+		twitter: "bjorn_fahller",
+		github: "rollbear",
+        mastodon: "https://mastodon.social/@bjorn_fahller"
+    },
+	pronouns: "",
+	profileImg: "./rollbear.jpg",
+	color: "#000000",
```

Comment:
Change this to your favorite color to add a bit of customization in the future! 😊